mantidproject/mantid
# modified database.load_calibration_set to clone and cast TableWorkspa…

…ce to MaskWorkspace

**Description of work.**

As per the ticket, it alters the Corelli database.py fucntion `load_calibration_set` to return a MaskWorkspace as opposed to a TableWorkspace.

It uses the instructions suggested in the ticket with some minor modifications.



**To test:**


The ticket contains the testing details which includes data files and a python test script.  
If you are running it from the terminal I suggest adding the line `print(type(mask))` at the end to confirm it is a workspace of the correct type.

**Fair warning**: I am a new developer on the project and lack a good chunk of context for my actions.  Please examine carefully.

I can tell the type is correct but I am unsure of how to confirm if the data is correct in the instance.

Fixes #136 (https://code.ornl.gov/sns-hfir-scse/diffraction/single-crystal/single-crystal-diffraction/-/issues/136)

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
